ArduPlane/events.cpp

```diff
@@ -22,7 +22,6 @@ void Plane::failsafe_short_on_event(enum failsafe_state fstype, ModeReason reaso
 {
```

Comment:
maybe `rc_failsafe_short_on_event`.

Reply:
also don't need to pass the mode reason or state, its always `FAILSAFE_SHORT` and `ModeReason::RADIO_FAILSAFE`